Dockerfile.iso

```diff
@@ -1,10 +1,16 @@
-FROM registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-node-image/5.2:latest AS os
-FROM registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-builder-image as tools
+ARG OS_IMAGE=registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-node-image/5.2:latest
+ARG TOOL_IMAGE=registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-builder-image
```

Comment:
I think that without `:lastest` at the end it's anyway the latest one by default?

Reply:
Yep, but we don't care, this repos are used to build an image with the obs published images and by the get-elemental-iso script that can be used from outside this repo so we want to default to the obs images, the official ones.

Alia I don't think we currently publish tags for those images, there is only one published at a time